app/views/landing_page/_hero.erb

```diff
@@ -19,11 +19,18 @@
   }
 <% end %>
 
+<%
+# Add non-breaking space to prevent widows
+subtitle_splitted = s["subtitle"]["value"].split(" ")
```

Comment:
Can the text be split to another line from some other character than space, e.g. hyphen?